package.json

```diff
@@ -6,7 +6,8 @@
     "lint-js-css": "stylelint --config '.stylelintrc-jsx.json' 'app/**/*.{ts,tsx}'",
     "lint-ts": "tslint --project tsconfig.json 'app/javascript/**/*.{ts,tsx}'",
     "prettier": "prettier --check 'app/javascript/**/*.{ts,tsx}'",
-    "test": "jest"
+    "test": "jest",
+    "build": "cd public/creators-landing && yarn install && yarn build"
```

Comment:
Heroku's build platform will run `yarn build` during deployments and thus our creators-landing page is going to be built when we deploy to ensure that regardless if someone hasn't committed the latest built files the website will function as intended.